dbms/tests/queries/0_stateless/01037_polygon_dict.sql

```diff
@@ -0,0 +1,70 @@
+SET send_logs_level = 'none';
+
+DROP DATABASE IF EXISTS test_01037;
+
+CREATE DATABASE test_01037 Engine = Ordinary;
+
+DROP DICTIONARY IF EXISTS test_01037.dict;
+DROP TABLE IF EXISTS test_01037.polygons;
+
+CREATE TABLE test_01037.polygons (key Array(Array(Array(Array(Float64)))), name String, value UInt64) ENGINE = Memory;
+INSERT INTO test_01037.polygons VALUES ([[[[1, 3], [1, 1], [3, 1], [3, -1], [1, -1], [1, -3], [-1, -3], [-1, -1], [-3, -1], [-3, 1], [-1, 1], [-1, 3]]], [[[5, 5], [5, 1], [7, 1], [7, 7], [1, 7], [1, 5]]]], 'Click', 42);
```

Comment:
I think it's better to use tuples points, not arrays.

Reply:
Also, it may be continent to accept polygons without holes as single-level arrays of tuples.

Reply:
Added both options.